examples/visual-tests/amp-story/amp-story-page-attachment-v2.html

```diff
@@ -0,0 +1,189 @@
+    <style amp-custom>
+      amp-story-page {
+        background: linear-gradient(#de6262, #ffb88c);
+        color: white;
+        font-family: sans-serif;
+      }
+
+      .i-amphtml-story-spinner-container {
+        /* Hide the spinner for loading pages, since this causes flakes */
+        display: none;
+      }
+
+      amp-story-page-attachment {
+        color: rgba(0, 0, 0, 0.87);
+        font-size: 16px;
+        line-height: 1.48em;
+      }
+
+      amp-story-page-attachment h1 {
+        padding: 0 24px;
+        font-size: 28px;
+        line-height: 1.48em;
+      }
+
+      amp-story-page-attachment p {
+        padding: 0 24px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <amp-story standalone publisher="AMP Team" title="Visual Diff Test"
+        publisher-logo-src="/examples/visual-tests/photos/120.png"
+        poster-portrait-src="/examples/visual-tests/picsum.photos/image981_900x1600.jpg"
+        poster-landscape-src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg"
+        poster-square-src="/examples/visual-tests/picsum.photos/image981_1600x1600.jpg"
+        supports-landscape>
+      <amp-story-page id="cover" id="inline-default">
+        <amp-story-page-attachment layout="nodisplay">
+          <h1>Some really cool video</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="page1" id="inline-custom-text">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          cta-text="Custom text">
+          <h1>Some really cool video</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="page2" id="inline-dark-theme">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          theme="dark">
+          <h1>Some really cool video</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="page3" id="inline-dark-theme-1-image">
```

Comment:
[Elements can only have one `id`](https://www.w3schools.com/html/html_id.asp).
Let's keep the descriptive id and use that to get the page in the tests.